VKClientRM/VKClientRM/Controllers/SearchGroupTableViewController.swift

```diff
@@ -57,7 +57,7 @@ final class SearchGroupTableViewController: UITableViewController {
             ) as? SearchGroupTableViewCell,
             indexPath.row < groups.count
         else { return UITableViewCell() }
-        cell.configureCell(group: groups[indexPath.row])
+        cell.configureCell(group: groups[indexPath.row], vkNetworkService: VKNetworkService())
```

Comment:
```suggestion
        cell.configureCell(group: groups[indexPath.row], vkNetworkService: VKNetworkService)
```
 а экземпляр класса создаем выше в контроллере